extensions/amp-consent/0.1/cmps.js

```diff
@@ -71,6 +71,12 @@ CMP_CONFIG['SourcePoint'] = {
   'promptUISrc': 'https://amp.pm.sourcepoint.mgr.consensu.org/',
 };
 
+CMP_CONFIG['opencmp'] = {
+  'consentInstanceId': 'opencmp',
+  'checkConsentHref': 'https://amp.opencmp.net/consent/check',
```

Comment:
I've been trying to test this locally, and it seems like http://localhost:8000 as an allowed origin. 

Would you mind adding localhost to you whitelist so that I can verify that the endpoint works with amp-consent?

More info here: https://amp.dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests/

Reply:
Die API is using the Origin header to send it back in the Access-Control-Allow-Origin header, so it should work. My local setup is working fine. Please check again.
If there are still problems, please send me more details about your setup and the request headers.

Reply:
Ok, thank you. I am getting a response now :)